### PR TITLE
GNOME 47

### PR DESCRIPTION
**Pull request by oscfdezdz**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 .tint {
   /* Dark view_bg_color */
-  background-color: rgba(30, 30, 30, 1);
+  background-color: rgb(30 30 30 / 100%);
```
Comment (DaPigGuy) [inline src/style.css:19]:
```suggestion
  background-color: rgb(30 30 30);
```

Comment (oscfdezdz) [reply src/style.css]: Thank you! It's fixed now.

**Comment by oscfdezdz** [comment]
Checking https://developer.gnome.org/hig/patterns/containers/boxed-lists.html#adding-buttons, it looks like the search page is the only one where the switch to AdwButtonRow is correct.

**Comment by oscfdezdz** [comment]
This should be ready for broader testing, I'm going to merge it. It only remains to change the `text-engine` module to the next stable version when it becomes available.

Note: master runtime breaks updates banner position, 47 runtime works as expected.

**Pull request merged**